### ISSUE TITLE
Verify confirmation mailer text is working as intended 

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#2129]
It looks like we just use Spotlight's default confirmation mailer text when sending the confirmation email to newly-added exhibit users. The text is pretty generic (which is fine for Spotlight core), which results in an email that can resemble spam. 

https://github.com/projectblacklight/spotlight/blob/66776d7caa5f984246064e2dc33422af7fa734c3/config/locales/spotlight.en.yml#L409

If we customize this text in Exhibits I think we can make the email feel more legitimate.

**Comment by ggeisler** [comment]
It looks like there might be more to this than I thought. Running Exhibits locally (I was going to update the text as suggested above) I added a new user to an exhibit and got the confirmation email below. This is a very different, customized version of the email than @caaster reported in #2128. (It also confirms my vague feeling that we had already written up customized text for the confirmation email.)

<img width="757" alt="Screen Shot 2021-09-13 at 5 33 45 PM" src="https://user-images.githubusercontent.com/101482/133175112-2ceec395-0bee-4026-bea1-0fefcbb3b75f.png">

---

Compare this to the example @caaster provided:

![Screen Shot 2021-09-13 at 3 16 57 PM](https://user-images.githubusercontent.com/3210225/133163831-0a192204-f716-440e-b2e3-7ff653109c09.png)

---

Are we somehow not using the customized version in production? Or am I missing something and we have different confirmation emails for different purposes?

**Comment by ggeisler** [comment]
Further investigation reveals that this is a different confirmation email than the one I referenced above. That was the confirmation a new user added as an exhibit admin/curator receives. The email in question is one a user receives when they are added as a feedback form recipient. 

I've created https://github.com/projectblacklight/spotlight/issues/2779 to focus on updating the text of the feedback form recipient confirmation email, which makes this ticket no longer necessary.
